bot.py

```diff
@@ -0,0 +1,101 @@
+from itertools import cycle
+from os import listdir
+import os
+from discord.ext import commands, tasks
+from discord import Intents
+from discord.ext.commands import Bot
+import discord
+from dotenv import load_dotenv
+import logging
+
+intents = Intents().all()
+
+load_dotenv()
+
+TOKEN = os.getenv('DISCORD_TOKEN')
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def get_prefix(bot, message):
+    """This function returns a Prefix for our bot's commands.
+
+    Args:
+        bot (commands.Bot): The bot that is invoking this function.
+        message (discord.Message): The message that is invoking.
+
+    Returns:
+        string or iterable containing strings: A string containing prefix or an iterable containing prefixes
+    Notes:
+        Through a database (or even a json) this function can be modified to returns per server prefixes.
+        This function should returns only strings or iterable containing strings.
+        This function shouldn't returns numeric values (int, float, complex).
+        Empty strings as the prefix always matches, and should be avoided, at least in guilds.
+    """
+    if not isinstance(message.guild, discord.Guild):
+        """Checks if the bot isn't inside of a guild. 
+        Returns a prefix string if true, otherwise passes.
+        """
+        return '!'
+
+    return ['!', '?', '>']
+
+
+# bot = commands.Bot(command_prefix=get_prefix, description='SuperstonkQuantBot Commands', intents=intents)
+bot = Bot(command_prefix=get_prefix, description='SuperstonkQuantBot Commands', intents=intents)
```

Comment:
Is this just for the commands? So you don't have to have the exact capitalization when typing the command?

Reply:
yes, otherwise you'd have to match the capitalization of the command. if you comman would be !role, a !Role would not trigger the command. 

`bot = Bot(command_prefix=get_prefix, description='SuperstonkQuantBot Commands', intents=intents, case_insensitive=True)`